doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -597,6 +599,58 @@ Finally, we can upgrade to our target version 7.3.x as follows.
 apt upgrade xroad-centralserver
 ```
 
+### 5.6 Data quality issues in the database
+
+The following error message may come up during the Central Server upgrade.
+
+`Data quality issues in the XXXXX database. ...`
+
+Before upgrading the packages from the current version to the target version first need to fix incorrect data in the postgreSQL database.
+
+For example, if the error message says:
+
+```bash
+root@test-cs:~# apt upgrade xroad-centralserver
+...
+Preparing to unpack .../xroad-center_7.4.0-1.ubuntu22.04_all.deb ...
+
+ERROR: Data quality issues in the centerui_production database. There are duplicate data in the table SYSTEM_PARAMETERS columns pair (KEY, HA_NODE_NAME):.........................................................] 
+----------------------------------------------------------------------------------------------------------------------------------------------
+id|key|value|created_at|updated_at|ha_node_name|count
+12|instanceIdentifier|cs|2021-03-10 07:37:26.59307|2021-03-10 07:37:26.59307|node_0|2
+1|instanceIdentifier|CS|2021-03-10 07:37:26.59307|2021-03-10 07:37:26.59307|node_0|2
+(2 rows)
+----------------------------------------------------------------------------------------------------------------------------------------------
+...
+Please correct incorrect data before continue.
+```
+
+To fix incorrect data, login to the postreSQL database server as the superuser (postgres by default):
+
+```bash
+psql -h <database host> -U <superuser> -d <database>
+```  
+
+and run script that shows all duplicate data in the SYSTEM_PARAMETERS table.
+
+```sql
+select * from (select *, count(*) over (partition by key, ha_node_name order by key, ha_node_name) as count
+      from system_parameters) as s
+where s.count > 1;
+```
+
+If script return data, then delete thus system parameters rows what system parameter value is not used or just delete one duplicate row if the values is same.
+
+```sql
+delete from system_parameters where id = 12;
+```
+
+The interrupted installation can be finished using:
+
+```bash
+sudo apt -f install
```

Comment:
It would be clearer to write long option instead of `-f`

```suggestion
sudo apt --fix-broken install
```
or (I'm not 100% sure if apt supports it in this order)
```suggestion
sudo apt install --fix-broken
```